docs/_docs/controls.md

```diff
@@ -180,6 +181,16 @@ float min = options.getPreviewFrameRateMinValue();
 float max = options.getPreviewFrameRateMaxValue();
 ```
 
+##### cameraPreviewFrameRateExact
+Controls the behavior of `cameraPreviewFrameRate`. If this option is set to `true`, the narrowest
+range containing the new preview fps will be used. If this option is set to `false` the broadest
+range containing the new preview fps will be used.
```

Comment:
Thanks for this! I guess I would appreciate a comment here about the fact that cameraPreviewFrameRateExact=true is a bit risky like I said before.